File: Locket/src/main/java/com/locket/payment/domain/pay/dto/PaymentRequest.java
package com.locket.payment.domain.pay.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class PaymentRequest {
    private int cardId;
    private long sellerId;
    private String paymentCategory;
    private String paymentMerchant;
    private BigDecimal amount;
    private String storeName;
    private String paymentKey; // ✅ 중복 방지 키 추가
}

File: Locket/data/src/main/java/com/ssafy/locket/data/network/request/payment/PaymentRequest.kt
package com.ssafy.locket.data.network.request.payment

import java.math.BigDecimal

data class PaymentRequest(
    val amount: BigDecimal,
    val cardId: Int,
    val paymentCategory: String,
    val paymentKey: String,
    val paymentMerchant: String,
    val sellerId: Long,
    val storeName: String
)